config.proto

```diff
@@ -29,8 +29,8 @@ message AgentConfig {
 // Reporting covers the options related to the mechanics for sending data to the
 // tracing server o collector.
 message Reporting {
-    // address represents the host for reporting the traces e.g. api.traceable.ai
-    google.protobuf.StringValue address = 1;
+    // endpoint represents the endpoint for reporting the traces e.g. http://api.traceable.ai:9411/api/v2/spans
+    google.protobuf.StringValue endpoint = 1;
 
     // when `true`, connects to endpoints over TLS.
     google.protobuf.BoolValue secure = 2;
```

Comment:
Since we changed the endpoint to define the protocol, is the `secure` flag still needed? cc @davexroth @mohit-a21

Reply:
unfortunately, I think it is - for the grpc case where a host:port is all that is specified for the endpoint.